website/blog/2025-04-18-dbt-mcp-server.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Introducing the dbt MCP Server – Bringing Structured Data to AI Workflows and Agents"
+description: "We’re open‑sourcing an experimental dbt MCP server so LLMs and agents can discover, query, and run your dbt project."
+slug: introducing-dbt-mcp-server
+authors: [jason_ganz]
+tags: [ai, data ecosystem]
+hide_table_of_contents: false
+date: 2025-04-21
+is_featured: true
+---
+
+# Introducing the dbt MCP Server - Bringing Structured Data to AI Workflows and Agents
+
+dbt is the standard for creating governed, trustworthy datasets on top of your structured data. [MCP](https://www.anthropic.com/news/model-context-protocol) is [showing](https://techcrunch.com/2025/04/09/google-says-itll-embrace-anthropic-standard-for-connecting-ai-models-to-data/) [increasing](https://techcrunch.com/2025/03/26/openai-adopts-rival-anthropic-standard-for-connecting-ai-models-to-data/) [promise](https://www.microsoft.com/en-us/microsoft-copilot/blog/copilot-studio/introducing-model-context-protocol-mcp-in-copilot-studio-simplified-integration-with-ai-apps-and-agents/) as the standard for providing context to LLMs to allow them to function at a high level in real world, operational scenarios.
```

Comment:
Links at showing and increasing not working. It's also not obvious in the ui that you're linking to 3 different articles.